spec/controllers/blocks_controller_spec.rb

```diff
@@ -0,0 +1,198 @@
+require 'rails_helper'
+
+RSpec.describe BlocksController, type: :controller do
+  describe "GET index" do
+    it "requires login" do
+      get :index
+      expect(response).to redirect_to(root_url)
```

Comment:
Does this get a flash? Should we test that it has that flash?

Reply:
The flash is a temporary 'this is incomplete' notice, so no.